src/lib/php/common-perms.php

```diff
@@ -147,7 +147,7 @@ function GetGroupUsers($user_pk, $group_pk, &$GroupArray)
    *  -  [group_name]
    *  -  [group_perm]
    **/
-function GetUsersGroups($user_pk='')
+function GetUsersGroups($user_pk=''): array
```

Comment:
The function does not always return. So, the value can also be null.
```suggestion
function GetUsersGroups($user_pk=''): ?array
```